.github/workflows/dev-aws-ecr.yml

```diff
@@ -0,0 +1,69 @@
+name: Docker Stuff
```

Comment:
😂  do you mind giving these a little more descriptive names? perhaps `Docker build main`, `Docker build alpha`?

Reply:
I don't mind at all!! Good idea!

Reply:
Hahahaha, @esacteksab, you name things the exact same way I do!